apps/publisher/themes/default/js/popover.js

```diff
@@ -43,6 +43,33 @@ var popoverFunction = (function(){
     return returnObj;
 })();
 
+var currentPageFunction = (function(){
```

Comment:
Thanks, I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/38e71195e4ea971e38601e6537597a13e9c24459)